content/browser/frame_host/cross_process_frame_connector.cc

```diff
@@ -410,6 +413,13 @@ void CrossProcessFrameConnector::EmbedRendererWindowTreeClientInParent(
       base::BindOnce(&RenderWidgetHostViewBase::OnChildFrameDestroyed,
                      parent->GetWeakPtr(), frame_routing_id));
 }
+
+void CrossProcessFrameConnector::OnSetFrameSinkId() {
+  CHECK(switches::IsMusHostingViz());
```

Comment:
minor: I think a DCHECK would be ok here too (although CHECK isnt wrong).

Reply:
done